charts/zigbee2mqtt/values.yaml

```diff
@@ -8,8 +8,8 @@ image:
 strategy:
   type: Recreate
 
-env: {}
-  # TZ
+env:
+  ZIGBEE2MQTT_DATA: /data
```

Comment:
It's good to have this default to gain persistence, but it can't start now since the /data folder is not enabled by default. 

Maybe set the defaults for `persistence.data.enabled` and `persistence.data.emptyDir`to `true`?

Reply:
Maybe we can add it in the ct.yaml as an override for the ci while keeping it disabled in these values?